dipy/denoise/gibbs.py

```diff
@@ -293,12 +293,11 @@ def gibbs_removal(vol, slice_axis=2, n_points=3, inplace=True, num_threads=1):
     if not isinstance(inplace, bool):
         raise TypeError("inplace must be a boolean.")
 
-    if (not isinstance(num_threads, int)) and (num_threads is not None):
-        raise TypeError("num_processes must be an int or None.")
+    if not isinstance(num_threads, int):
+        raise TypeError("num_threads must be an int.")
     else:
-        if isinstance(num_threads, int):
-            if num_threads <= 0:
-                raise ValueError("num_processes must be > 0.")
+        if num_threads < 0:
+            raise ValueError("num_threads must be >= 0.")
```

Comment:
I think, you do not need this.  `-2`, `-1` or `0` should be the same: All core use. it gives a bit more flexiblity